Source/GameBaseFramework/Private/Gameplay/ConditionalEvents/GBFConditionalEventSubsystem.cpp

```diff
@@ -0,0 +1,137 @@
+#include "Gameplay/ConditionalEvents/GBFConditionalEventSubsystem.h"
+
+#include "GameFramework/GBFGameState.h"
+#include "Gameplay/ConditionalEvents/GBFConditionalEvent.h"
+#include "Gameplay/ConditionalEvents/GBFConditionalEventGroupData.h"
+
+#include <AbilitySystemComponent.h>
+
+bool UGBFConditionalEventSubsystem::ShouldCreateSubsystem( UObject * outer ) const

```

Comment:
On Lyra, the phase subsystem does not implement that function. The system would be created everywhere. But you need to add `BlueprintAuthorityOnly` on the BP functions

Reply:
Done.